src/XIVLauncher.Core/Components/SettingsPage/Tabs/SettingsTabWine.cs

```diff
@@ -54,7 +54,6 @@ public SettingsTabWine()
             },
             new SettingsEntry<string>("DXVK Frame Limit", "Configure how many frames DXVK should be limited to. Set to 0 for unlimited.", () => Program.Config.DxvkFrameRate ?? "0", s => Program.Config.DxvkFrameRate = s)
             {
-                CheckVisibility = () => RuntimeInformation.IsOSPlatform(OSPlatform.Linux),
```

Comment:
After discussion with marzent about DXVK on Windows, I've opted to remove Linux-only visibility/